### PR TITLE
Calcite fixup for junit

### PR DESCRIPTION
**Pull request by kaikaizi**
Going through JUnit tests to make sure that all changes made, or tests skipped on calcite integration branch should still persist if git branch name contains "**calcite_**"; but should otherwise behave the same way as master branch.

### CODE REVIEW

File: tests/frontend/org/voltdb/regressionsuites/TestFunctionsForVoltDBSuite.java [hunk 1]
```diff
@@ -919,92 +918,125 @@ public void testSINCE_EPOCH() throws Exception {
...
             result = cr.getResults()[0];
             assertEquals(1, result.getRowCount());
             assertTrue(result.advanceRow());
-            if (proc == "SINCE_EPOCH_SECOND") {
-                assertEquals(0, result.getLong(0));
-            } else if (proc == "SINCE_EPOCH_MILLIS" || proc == "SINCE_EPOCH_MILLISECOND") {
-                assertEquals(0, result.getLong(0));
-            } else if (proc == "SINCE_EPOCH_MICROS" || proc == "SINCE_EPOCH_MICROSECOND") {
-                assertEquals(0, result.getLong(0));
+            switch (proc) {
```
Comment (GuaGuaDean) [inline tests/frontend/org/voltdb/regressionsuites/TestFunctionsForVoltDBSuite.java]: We can git rid of this switch statement

File: tests/frontend/org/voltdb/regressionsuites/TestFunctionsSuite2.java [hunk 2]
```diff
@@ -246,32 +250,37 @@ DESC VARCHAR(300),
...
+                "select -ratio from P1 where desc='maxvalues'");
         assertEquals(ClientResponse.SUCCESS, cr.getStatus());
         r = cr.getResults()[0];
         r.advanceRow();
         // actually returns NULL but when we do r.get("C1", VoltType.FLOAT) it returns more precision
         assertTrue( (Double) r.get("C1", VoltType.FLOAT) <= VoltType.NULL_FLOAT );
 
         // testing the same behavior for 0-Double.MAX_VALUE
-        cr = client.callProcedure("@AdHoc", "select 0-ratio from P1 where desc='maxvalues'");
+        cr = client.callProcedure("@AdHoc","select 0-ratio from P1 where desc='maxvalues'");
```
Comment (GuaGuaDean) [inline tests/frontend/org/voltdb/regressionsuites/TestFunctionsSuite2.java]: Space should be kept between params

File: tests/frontend/org/voltdb/regressionsuites/TestFunctionsSuite2.java [hunk 3]
```diff
@@ -246,32 +250,37 @@ DESC VARCHAR(300),
...
         r.advanceRow();
         // actually returns NULL but when we do r.get("C1", VoltType.FLOAT) it returns more precision
         assertTrue( (Double) r.get("C1", VoltType.FLOAT) <= VoltType.NULL_FLOAT );
 
-        client.callProcedure("@AdHoc", "INSERT INTO P1 VALUES (4, 'minvalues', 0 , " + Double.MIN_VALUE + " , NULL)");
+        client.callProcedure("@AdHoc",
+                "INSERT INTO P1 VALUES (4, 'minvalues', 0 , " + Double.MIN_VALUE + " , NULL)");
 
-        cr = client.callProcedure("@AdHoc", "select -ratio from P1 where desc='minvalues'");
+        cr = client.callProcedure("@AdHoc","select -ratio from P1 where desc='minvalues'");
```
Comment (GuaGuaDean) [inline tests/frontend/org/voltdb/regressionsuites/TestFunctionsSuite2.java]: Space ..

File: tests/frontend/org/voltdb/regressionsuites/TestFunctionsSuite2.java [hunk 4]
```diff
@@ -312,21 +323,19 @@ DESC VARCHAR(300),
         assertEquals( rA.get( "C5", VoltType.FLOAT), rB.get( "C5", VoltType.FLOAT ));
         assertEquals( rA.get( "C6", VoltType.DECIMAL), rB.get( "C6", VoltType.DECIMAL ));
 
-        client.callProcedure("@AdHoc", "delete from NUMBER_TYPES where INTEGERNUM = 1");
+        client.callProcedure("@AdHoc","delete from NUMBER_TYPES where INTEGERNUM = 1");
```
Comment (GuaGuaDean) [inline tests/frontend/org/voltdb/regressionsuites/TestFunctionsSuite2.java]: Space ..

File: tests/frontend/org/voltdb/regressionsuites/TestFunctionsSuite2.java [hunk 5]
```diff
@@ -354,13 +364,17 @@ DESC VARCHAR(300),
...
+        client.callProcedure("@AdHoc",
+                "INSERT INTO P1 VALUES (2, 'wEoiXIuJwSIKBujWv', -29914, 8.98500019539639316335e-01, NULL)");
+        client.callProcedure("@AdHoc",
+                "INSERT INTO P1 VALUES (4, 'WCfDDvZBPoqhanfGN', -1309657, 9.34160160574919795629e-01, NULL)");
+        client.callProcedure("@AdHoc",
+                "INSERT INTO P1 VALUES (6, 'WCfDDvZBPoqhanfGN', 1414568, 1.14383710279231887164e-01, NULL)");
+        cr = client.callProcedure("@AdHoc","select (5.25 + NUM) from P1");
         assertEquals(ClientResponse.SUCCESS, cr.getStatus());
-        cr = client.callProcedure("@AdHoc", "SELECT FLOOR(NUM + 5.25) NUMSUM FROM P1 ORDER BY NUMSUM");
+        cr = client.callProcedure("@AdHoc","SELECT FLOOR(NUM + 5.25) NUMSUM FROM P1 ORDER BY NUMSUM");
```
Comment (GuaGuaDean) [inline tests/frontend/org/voltdb/regressionsuites/TestFunctionsSuite2.java]: Space ..

File: tests/frontend/org/voltdb/TestAdHocQueries.java [hunk 6]
```diff
@@ -191,21 +184,15 @@ public void testProcedureAdhoc() throws Exception {
...
             if (m_client != null) m_client.close();
             m_client = null;
 
-            if (localServer != null) {
-                localServer.shutdown();
-                localServer.join();
-            }
-            localServer = null;
-
+            localServer.shutdown();
```
[the remark this answers is not in the document]
Comment (kaikaizi) [reply tests/frontend/org/voltdb/TestAdHocQueries.java]: This is not about coding style. `localServer` can never be null, so the null check is redundant.

File: tests/frontend/org/voltdb/TestAdHocQueries.java [hunk 7]
```diff
@@ -665,24 +647,18 @@ public void testAdHocQueryForStackOverFlowCondition() throws IOException, Except
...
                 m_client.close();
             }
             m_client = null;
-
-            if (localServer != null) {
-                localServer.shutdown();
-                localServer.join();
-            }
-            localServer = null;
+            localServer.shutdown();
```
[the remark this answers is not in the document]
Comment (kaikaizi) [reply tests/frontend/org/voltdb/TestAdHocQueries.java]: This is not about coding style. `localServer` can never be null, so the null check is redundant.

File: tests/frontend/org/voltdb/regressionsuites/TestFixedSQLSuite.java [hunk 8]
```diff
@@ -489,10 +487,7 @@ public void testBoxedByteArrays() throws IOException, InterruptedException, Proc
             assertTrue(e.getMessage().contains(errMsg));
         }
 
-        String query =
-                String.format("select * from ENG_539");
-        results = client.callProcedure("@AdHoc", query).getResults();
-        System.out.println(results);
+        client.callProcedure("@AdHoc", "select * from ENG_539");
         }, "ENG_539");
```
[the remark this answers is not in the document]
Comment (kaikaizi) [reply tests/frontend/org/voltdb/regressionsuites/TestFixedSQLSuite.java]: See `runWithClient()` in L77.

File: tests/frontend/org/voltdb/regressionsuites/TestFunctionsForVoltDBSuite.java (hunk 1 above)
Comment (kaikaizi) [reply tests/frontend/org/voltdb/regressionsuites/TestFunctionsForVoltDBSuite.java]: No, apparently we need it.

Comment (GuaGuaDean) [reply tests/frontend/org/voltdb/regressionsuites/TestFunctionsForVoltDBSuite.java]: Each case does the same thing, so you don't need the switch

File: tests/frontend/org/voltdb/regressionsuites/TestFixedSQLSuite.java (hunk 8 above)
[the remark this answers is not in the document]
Comment (GuaGuaDean) [reply tests/frontend/org/voltdb/regressionsuites/TestFixedSQLSuite.java]: OK

File: tests/frontend/org/voltdb/TestAdHocQueries.java (hunk 6 above)
[the remark this answers is not in the document]
Comment (GuaGuaDean) [reply tests/frontend/org/voltdb/TestAdHocQueries.java]: Neither is `m_client`, right? The null check for m_client should be redundant, too

File: tests/frontend/org/voltdb/TestAdHocQueries.java (hunk 7 above)
[the remark this answers is not in the document]
Comment (GuaGuaDean) [reply tests/frontend/org/voltdb/TestAdHocQueries.java]: Same logic, null check for m_client is redundant, too

File: tests/frontend/org/voltdb/TestAdHocQueries.java (hunk 6 above)
[the remark this answers is not in the document]
Comment (kaikaizi) [reply tests/frontend/org/voltdb/TestAdHocQueries.java]: Yes, if you check the implementation of `createClient()` logic, you can see that it cannot be null.
But, this is less obvious to IDE, and to human mind; and `createClient()` could legitlly return `null`. Once we changed that implementation, we might get NPE here if we don't do null check in the tests.

**Pull request merged**